storage/mysql/map_storage_test.go

```diff
@@ -161,44 +160,43 @@ func TestMapSetGetMultipleRevisions(t *testing.T) {
 	defer db.Close()
 	s := prepareTestMapStorage(mapID, t)
 
-	numRevs := 3
-	values := make([]trillian.MapLeaf, numRevs)
-	for i := 0; i < numRevs; i++ {
-		values[i] = trillian.MapLeaf{
-			KeyHash:   keyHash,
-			LeafHash:  []byte(fmt.Sprintf("A Hash %d", i)),
-			LeafValue: []byte(fmt.Sprintf("A Value %d", i)),
-			ExtraData: []byte(fmt.Sprintf("Some Extra Data %d", i)),
-		}
-	}
-
-	for i := 0; i < numRevs; i++ {
+	for _, tc := range []struct {
+		rev  int64
+		leaf trillian.MapLeaf
+	}{
+		{0, trillian.MapLeaf{keyHash, []byte{0}, []byte{0}, []byte{0}}},
+		{1, trillian.MapLeaf{keyHash, []byte{1}, []byte{1}, []byte{1}}},
+		{2, trillian.MapLeaf{keyHash, []byte{2}, []byte{2}, []byte{2}}},
+		{3, trillian.MapLeaf{keyHash, []byte{3}, []byte{3}, []byte{3}}},
+	} {
+		// Write the current test case.
```

Comment:
This isn't equivalent to previous test as it no longer does all writes before reading back. It's unlikely but it could possibly hide bugs. On some Merkle tests we do them twice, all at once and then like this incrementally. Not sure it's worth it here though.